src/components/new-request/search.vue

```diff
@@ -17,8 +17,15 @@
           item-value="[group,value]"
           :menu-props="{ bottom: true, offsetY: true, maxHeight: 423 }"
           @change="setClearErrors(null); onRequestActionChange($event)"
+          return-object
         >
-          <template v-slot:item="{ item }">
+          <!-- FUTURE: use "selection" slot to format the selected business -->
+          <!-- <template #selection="{ item }">
+            <div class="font-weight-bold text-truncate">{{ item.text }}</div>
+            <div class="text-subtitle-1">{{ item.subtext }}</div>
+          </template> -->
```

Comment:
I played with this a little on Janis' and Ethan's request, and we decided to not pursue this yet, but I left the code here for the next dev.